apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/SurveyEditor.tsx

```diff
@@ -38,10 +39,12 @@ export default function SurveyEditor({
 
   useEffect(() => {
     if (survey) {
-      setLocalSurvey(survey);
+      // Create a deep copy of the survey object
+      const surveyCopy = _.cloneDeep(survey);
```

Comment:
Please create the survey copy using
```
const updatedSurvey = {...survey};
```
I would love to not get lodash in the frontend bundle :-)

Reply:
@mattinannt const updatedSurvey = {...survey}; 
if i use above line , its not working.

Reply:
@mattinannt You can just `import { cloneDeep } from "lodash"`. That way it'll just get tree shaked, right?

Reply:
@mattinannt any update on this issue

Reply:
@RajuGangitla why is it not working with
```
const updatedSurvey = {...survey};
```
?

We can also make a module import, yes, but if we don't need this because it's a built-in js feature, that would be the preferred solution ;-)

Reply:
@mattinannt JSON.parse(JSON.stringify(survey));        this might work let me try it